rcl/test/rcl/test_wait.cpp

```diff
@@ -41,3 +46,132 @@ TEST(CLASSNAME(WaitSetTestFixture, RMW_IMPLEMENTATION), test_resize_to_zero) {
   ret = rcl_wait_set_fini(&wait_set);
   ASSERT_EQ(RCL_RET_OK, ret) << rcl_get_error_string_safe();
 }
+
+// Some test cases for the waitset
```

Comment:
Is there a more appropriate comment you can put here, one that perhaps summarizes what particular API or behaviors of the waitset are being tested?